aalto-grades/base-repository
# Test cases for navigating main features

Test cases that:
- create a course
- Navigate to a course instance
- Create a new course instance
- Verify the content of edit attainments page.

Notable Bug Noticed during the construction of these tests:
- Creating course instance without base from SISU is not working in main, thus the test case for it stays skipped.
